tests/test_network_models.py

```
@@ -0,0 +1,212 @@
+from typing import Dict, Any
+from pydantic import ValidationError
+
+from isminet.models.network import (
+    DHCPConfiguration,
+    VLANConfiguration,
+    NetworkConfiguration,
+)
+from isminet.models.enums import DHCPMode, IGMPMode
+
+# Test data
+VALID_DHCP_CONFIG: Dict[str, Any] = {
```

Ruff (0.8.2)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)